collect_app/src/main/java/org/odk/collect/android/map/GoogleMapFragment.java

```diff
@@ -430,6 +430,9 @@ protected static Marker createMarker(GoogleMap map, MapPoint point, boolean drag
         // A Marker's position is a LatLng with just latitude and longitude
         // fields.  We need to store the point's altitude and standard
         // deviation values somewhere, so they go in the marker's snippet.
+        if (map == null) {
```

Comment:
Nit: Either add this null check above the descriptive comment, or make that comment javadoc.

Reply:
Done.